python/equistore-core/equistore/core/labels.py

```diff
@@ -303,6 +303,141 @@ def range(name: str, end: int) -> "Labels":
             values=np.arange(end, dtype=np.int32).reshape(-1, 1),
         )
 
+    def append(self, name: str, values: np.ndarray) -> "Labels":
+        """Append a new column to the end.
+
+        :param name: name of the new column
+        :param values: 1D array of values for the new column
```

Comment:
I demand this to be a 1D array because we are stacking a new column to a 2D array. If you think this seems inconsistent w.r.t. to creating a new Labels object (where we demand a 2D array as input).